cmd/operator-sdk/alpha/scorecard/cmd.go

```diff
@@ -15,33 +15,40 @@
 package scorecard
 
 import (
-	log "github.com/sirupsen/logrus"
-	"github.com/spf13/cobra"
-)
+	"log"
 
-var (
-	config   string
-	bundle   string
-	selector string
-	listAll  bool
+	scorecard "github.com/operator-framework/operator-sdk/internal/scorecard/alpha"
+	"github.com/spf13/cobra"
 )
 
 func NewCmd() *cobra.Command {
+	var (
+		config   string
+		bundle   string
```

Comment:
Are `bundle` or `list` used anywhere in this file?

Reply:
those are not used at this point, I expect to implement those flags in upcoming stories.